src/App.css

```diff
@@ -1,39 +1,340 @@
+:root {
+  --pale-grey: #f5f6fa;
+  --azure: #0290ff;
+  --black-two: #121212;
+}
+
 .App {
-  text-align: center;
+  display: flex;
```

Comment:
- [ ] This CSS property is used more than once, it would be nice to use a reusable class which can be reused on the affected components, kindly do so to stick to [best practices](https://speckyboy.com/good-bad-css-practices/)